core/templates/dev/head/services/ExplorationHtmlFormatterServiceSpec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for services for explorations which may be shared
+ * by both the learner and editor views
+ */
+
+describe('Exploration Html Formatter Service', function() {
+  beforeEach(module('oppia'));
+
+  beforeEach(inject(function($injector) {
+    ehfs = $injector.get('ExplorationHtmlFormatterService');
```

Comment:
Where is the variable declaration for this?

Reply:
Done